packages/dcos-integration-test/extra/test_remove_dockercfg_hook.py

```diff
@@ -0,0 +1,36 @@
+import uuid
+
+import pytest
+
+
+@pytest.fixture(scope='module')
+def auth_cluster(cluster):
+    if not cluster.dockercfg_hook_enabled:
+        pytest.skip("Skipped because not running against cluster with remove .dockercfg hook.")
+    return cluster
+
+
+def test_remove_dockercfg_hook(cluster):
```

Comment:
I do not understand what is going on here at all. The build procedure has been changed to always dump a .dockercfg file into the integration testing package. This test should always pass with this build step, right?